neoutils/batch_cypher_runner.go

```diff
@@ -81,7 +81,7 @@ func processCypherBatch(bcr *BatchCypherRunner, currentQueries []*neoism.CypherQ
 			}{}
 
 			if jsonErr := json.Unmarshal([]byte(neoErr.Message), &neoErrMsg); jsonErr != nil {
-				log.Printf("ERROR Got error trying to process Neo Error Message, error=%v\n", jsonErr)
+				log.Errorf("ERROR Got error trying to process Neo Error Message, error=%v", jsonErr)
```

Comment:
I'd use `log.WithError(err).Error("msg")` here, instead of embedding the err. I'd also remove the `ERROR` strings from the message where appropriate